### ISSUE TITLE
Version 0.4.0 - Release Checklist

### ISSUE THREAD

**Issue by paragonie-scott** [paragonie/sodium-plus#17]
Normally I'd drop this in the milestone description, but that's less visible than a new issue.

First, we need a new libsodium-wrappers release so we can stop including our fork in `package.json`. This is being tracked in https://github.com/jedisct1/libsodium.js/issues/214

~~Next, we need to update the documentation to reflect the changes since `v0.2.0`.~~

Finally, we should ensure TypeScript compatibility is maintained, either via `@types/sodium-plus` or a `.ts` file in our repository. A Twitter user named "Le suisse" [offered to help with this](https://twitter.com/el_suisse/status/1181285158348939264). I'll circle back with them later this week and see if they're still going to send a pull request. If not, I'll tackle it.

In short:

- [x] Upstream release (`v0.7.5` -> `???`)
- [x] Documentation updates
- [x] TypeScript #13 
- [x] Browserify Fixes #21 

Edit: decided to just get `v0.3.0` out tonight. We can do `v0.4.0` for these changes.

**Comment by paragonie-scott** [comment]
libsodium-wrappers v0.7.6 was released, now I just need to get a browser-compat bundle working and we can release v0.4.0 of sodium-plus.
